extension/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Ha11y</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section class="header">
+      <div class="logo">
+        <img src="HALLY.png" alt="Ha11y Logo" width="100px" height="100px" />
+      </div>
+      <div class="title">Ha11y</div>
+    </section>
+
+    <section class="content">
+      <div class="sub-title">What is Ha11y?</div>
+      <div class="description">
+        Ha11y allows users to discover whether or not a website will be
+        accessible to them before even opening the website. Ha11y Google Chrome
+        Extension scores and displays the accessibility score of websites
+        directly in a google search for users with disabilities. Users can
+        select their disability when first signing up and the results will be
+        tailored to them, otherwise, there is also a default score for general
+        accessibility.
+      </div>
+
+      <div class="sub-title">What applies to you?</div>
+
+      <div class="description">
+        <form>
+          <label for="blind">
+            <input type="checkbox" id="blind" name="blind" value="Blind" />
+            Blind</label
+          ><br />
```

Comment:
Odd formatting.

Reply:
that's how my auto formatter formatted it